Resources/Prototypes/_Lavaland/Construction/explorer_suit.yml

```
@@ -0,0 +1,30 @@
+            amount: 3
+            doAfter: 2
+    - node: reinforced2
+      entity: LavalandEquipmentExplorerSuitReinforced2
+      edges:
+      - to: reinforced3
+        steps:
+          - material: GoliathHide
+            amount: 3
+            doAfter: 2
+    - node: reinforced3
+      entity: LavalandEquipmentExplorerSuitReinforced3
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)